zh/upgrade-a-tidb-cluster.md

```diff
@@ -125,3 +125,7 @@ helm upgrade ${release_name} pingcap/tidb-cluster -f values.yaml --version=${cha
 > ```shell
 > kubectl annotate tc ${release_name} -n ${namespace} tidb.pingcap.com/force-upgrade-
 > ```
+
```

Comment:
Is it better we move this warning to the beginning of this doc, e.g. L16?

Reply:
It's a note that user can know but will not cause harm when user is not aware of.

Reply:
We are also following [CRDB's way](https://www.cockroachlabs.com/docs/stable/install-cockroachdb-mac.html) to put this kind of friendly note at a secondary place.